src/favouritesPage/components/favouriteCard.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import { Button, Card, LoadingSpinner } from 'hds-react';
+import { Row, Col } from 'react-grid-system';
+import { IconCrossCircle, IconArrowDown, IconArrowUp } from 'hds-react';
+import { useTranslation } from 'react-i18next';
+import { PlotSearch, PlotSearchTarget } from '../../plotSearch/types';
+import { getInfo } from '../utils';
+import { FavouriteCardDetails } from './favouriteCardDetails';
+import FavouriteCardMap from './favouriteCardMap';
+
+interface Props {
+  target: PlotSearchTarget;
+  remove: (id: number) => void;
+  plotSearch: PlotSearch | null;
+}
+
+const FavouriteCard = (props: Props): JSX.Element => {
+  const { target, remove, plotSearch } = props;
+  const { t } = useTranslation();
+  const [fullDescription, setFullDescription] = useState(false);
+  if (!plotSearch) {
+    return (
+      <Col className="Favourite">
+        <LoadingSpinner />
+      </Col>
+    );
+  }
+
+  const infoCols = getInfo(target, plotSearch, t);
+
+  return (
+    <Card className="FavouriteCard" border key={target.id}>
+      <Row>
+        <Col md={3} xs={4}>
+          {/* Left Column (map)*/}
+          <FavouriteCardMap target={target} />
+        </Col>
+        <Col md={9} xs={8}>
+          {/* Right Column (text and actionbuttons etc.. */}
+          <Row className="FavouriteCard__header-row">
+            {/* Top row with title and delete action on top right */}
+            <Col md={8} xs={7}>
+              <h3>{target.lease_address.address}</h3>
+            </Col>
+            <Col md={4} xs={5}>
+              <Button
+                className="FavouriteCard__action-button"
+                onClick={() => remove(target.id)}
+                variant="supplementary"
+                iconLeft={
+                  <IconCrossCircle className="FavouriteCard__action-button__icon" />
```

Comment:
One more tweak: there's no further hierarchy in BEM below the block level, so all elements should be treated as equal. Meaning that there's no B__E__E, just B__E and this can thus be just `FavouriteCard__action-button-icon`. Same with the later occurrences as well.